config/nginx/README.md

```diff
@@ -0,0 +1,5 @@
+## Where to put the configuration files?
+- default -> /etc/nginx/sites-available/default
+
+## Other actions required
+None
```

Comment:
What about changing persmissions and ownership to the website directory (/var/www/html)? I had "403 Forbidden" error without this:
`chmod -R 775 /var/www/html/`
`chown -R www-data:www-data /var/www/html/`
Does it work without it on your machine?

Reply:
I did it a bit differently, but you're right that it should be applied.